examples/jsm/nodes/materials/NodeMaterial.js

```diff
@@ -144,7 +145,7 @@ class NodeMaterial extends ShaderMaterial {
 
 			const alphaTestNode = this.alphaTestNode ? float( this.alphaTestNode ) : materialAlphaTest;
 
-			stack.add( diffuseColor.a.lessThanEqual( alphaTestNode ).discard() );
+			stack.add( discard( diffuseColor.a.lessThanEqual( alphaTestNode ) ) );
```

Comment:
Strange I'll have to investigate `discard( node )` should behave like `node.discard()`

Reply:
Could you make a step by step to reproduce this problem? Currently this runs fine in `webgpu_materials.html` example.

https://github.com/mrdoob/three.js/blob/aef215178070679b8407832a9a71123978807456/examples/webgpu_materials.html#L129-L134

Same behavior with `material.alphaTest`

![image](https://github.com/mrdoob/three.js/assets/502810/1d917516-026f-4033-b28a-dd3f02dfea72)